packages/icon-library/package.json

```diff
@@ -13,6 +13,11 @@
   ],
   "types": "dist/types/index.d.ts",
```

Comment:
I was under the assumption that `main` and `types` were enough, so I'm surprised it's required to respect the ESM output. Does it supersede these - should we get rid of them?

Reply:
Just read up on it looks like it's common to keep them as fallbacks for older tooling.